grifts/shoulders.go

```diff
@@ -1,14 +1,14 @@
 package grifts
 
 import (
-	"fmt"
 	"html/template"
 	"os"
 	"os/exec"
 	"path"
 	"sort"
 	"strings"
 
+	"github.com/Sirupsen/logrus"
```

Comment:
The repo is now lower case `sirupsen`. Can you please undo these import changes?